### ISSUE TITLE
[Request] Add Modified Exploration/SM Maps

### ISSUE THREAD

**Issue by pikangie** [cursey/kanan#159]
I have been trying to look for a working mod of these modified maps, but I could not get any of them to work with Kanan. 
I also tried installing a modified version of Tiara/Abyss after hearing it had the Exploration maps, but I really cannot stand the default FOV which those mods forced back again... Something about the Tiara/Abyss mods caused the Mabinogi game to be unusable anymore when I tried to run it with Kanan and also later when I uninstalled Tiara/Abyss.

In short, I would be perfectly happy with using only Kanan over Tiara or Abyss, if it just had the revealed exploration maps and glowing L-rod mods.

PS: Oh yeah, also I liked the dye ampoules that were displayed with the Tiara/Abyss mod, maybe that is easier to add as well?

**Comment by cursey** [comment]
The maps should just be a data folder mod which kanan supports if you have `UseDataFolder.js` enabled. You might need to unpack any of the .pack files included with tiara if that is how its being packaged now-a-days to find the map textures. 

Then all you have to do is mimic the structure in kanan's `/data/` folder.

I can write better instructions when I find the time.

**Comment by pikangie** [comment]
It doesn't seem to work. @_@ Not really sure why, but I am following the same data path as where the original instructions tell me to put it, and copied it the same way in the Kanan folder as well. Maybe it needs something additional, or a Nexon update is blocking it? (Sorry I am new to this)

I put the .jpg files into the "data\gfx\gui\map_jpg" folder of both Mabinogi and Kanan Standalone directories.
